mozci/console/commands/push.py

```diff
@@ -225,12 +226,54 @@ def handle(self) -> None:
                 "<comment>Provided --output pointed to a inexistent directory that is now created.</comment>"
             )
 
+        try:
+            backfill_limit = int(self.option("backfill-limit"))
+        except ValueError:
+            raise Exception("Provided --backfill-limit should be an int.")
+
         for push in pushes:
             try:
-                classification, regressions = push.classify(**classify_parameters)
+                classification, regressions, to_retrigger_or_backfill = push.classify(
```

Comment:
There is a problem to solve: if we run the script and retrigger something, we don't want to retrigger again on the next run if the previously retriggered tasks haven't finished yet.
We should make sure not to retrigger if there is any task in progress that is running the same groups (similar logic as #697).

Reply:
Yeah, let's do #697 first.

Reply:
Or better, let's implement `is_group_running` first, we don't have to fix the full #697.

Reply:
Will work on it early next week hopefully. :crossed_fingers: